core/src/main/java/org/sonarsource/sonarlint/core/container/connected/update/RulesDownloader.java

```diff
@@ -47,28 +44,31 @@
 import org.sonarsource.sonarlint.core.util.FileUtils;
 
 public class RulesDownloader {
+  static final String MSG_NO_DESC = "Rule descriptions are only available in SonarLint with SonarQube 5.1+";
   private static final org.sonarqube.ws.Rules.Active.Param.Builder PARAM_BUILDER = Param.newBuilder();
   private static final org.sonarqube.ws.Rules.Active.Builder AR_BUILDER = Active.newBuilder();
   private static final org.sonarqube.ws.Rules.ActiveList.Builder ARL_BUILDER = ActiveList.newBuilder();
   private static final org.sonarqube.ws.Rules.Rule.Builder RULE_BUILDER = Rule.newBuilder();
-  private static final String RULES_SEARCH_URL = "/api/rules/search.protobuf?f=repo,name,severity,lang,internalKey,isTemplate,templateKey,"
+  private static final String RULES_SEARCH_URL = "/api/rules/search.protobuf?f=repo,name,severity,lang,htmlDesc,internalKey,isTemplate,templateKey,"
     + "actives&statuses=BETA,DEPRECATED,READY";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "actives&statuses=BETA,DEPRECATED,READY" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1192)